.github/workflows/test.yml

```diff
@@ -0,0 +1,45 @@
+on: [push]
+name: Test
+jobs:
+  test:
+    name: Build and test driver on windows-2022
+    runs-on: windows-2019
+    env:
+      CASS_DRIVER_LIBEV_INCLUDES: C:/vcpkg/packages/libev_x64-windows/include/libev
+      CASS_DRIVER_LIBEV_LIBS: C:/vcpkg/packages/libev_x64-windows/lib
+      LIBUV_ROOT_DIR: C:/vcpkg/packages/libuv_x64-windows
+      ZLIB_ROOT_DIR: C:/vcpkg/packages/zlib_x64-windows-static
+      KERBEROS_ROOT_DIR: C:/vcpkg/packages/krb5_x64-windows
+      OPENSSL_ROOT_DIR: C:/vcpkg/packages/openssl_x64-windows
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ilammy/msvc-dev-cmd@v1
```

Comment:
Appears to be necessary to setup MSVC env properly.  More at [the Githubs](https://github.com/ilammy/msvc-dev-cmd).